src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/court/CourtService.kt

```diff
@@ -192,19 +192,19 @@ class CourtService(
         VideoLinkBookingResponse.LocationTimeslot(
           locationId = it.eventLocationId,
           startTime = it.startTime,
-          endTime = it.endTime
+          endTime = it.endTime!!
```

Comment:
This will throw an NPE if endTime is null

Reply:
This shouldn't ever be null as end time is required for Video link bookings. I cleaner way would be to have another model but seemed like over kill.

Reply:
Apparently never null for VLBs